independent-publisher-connectors/AzureADApplications/apiDefinition.swagger.json

```diff
@@ -81,405 +123,519 @@
+		  },
+		  "isFallbackPublicClient": {
+			"type": "boolean",
+			"description": "Specifies the fallback application type as public client, such as an installed application running on a mobile device. The default value is false which means the fallback application type is confidential client such as a web app"
+		  },
+		  "notes": {
+			"type": "string",
+			"description": "Notes relevant for the management of the application"
+		  },
+		  "optionalClaims": {
+			"description": "Optional claims in their Azure AD applications to specify the claims that are sent to their application by the Microsoft security token service"
+		  },
+		  "publisherDomain": {
+			"type": "string",
+			"description": "The verified publisher domain for the application"
+		  },
+		  "signInAudience": {
+			"type": "string",
+			"description": "Specifies the Microsoft accounts that are supported for the current application. Supported values are: AzureADMyOrg, AzureADMultipleOrgs, AzureADandPersonalMicrosoftAccount, PersonalMicrosoftAccount"
+		  },
+		  "tags": {
+			"type": "array",
+			"items": {},
+			"description": "Custom strings that can be used to categorize and identify the application"
+		  },
+		  "tokenEncryptionKeyId": {
+			"type": "string",
+			"description": "Specifies the keyId of a public key from the keyCredentials collection"
+		  },
+		  "verifiedPublisher": {
+			"type": "object",
+			"properties": {
+			  "displayName": {
+				"type": "string",
+				"description": "The verified publisher name from the app publisher's Partner Center account"
+			  },
+			  "verifiedPublisherId": {
+				"type": "string",
+				"description": "The ID of the verified publisher from the app publisher's Partner Center account"
+			  },
+			  "addedDateTime": {
+				"type": "string",
+				"description": "The timestamp when the verified publisher was first added or most recently updated"
+			  }
+			},
+			"description": "Specifies the verified publisher of the application"
+		  },
+		  "defaultRedirectUri": {
+			"type": "string",
+			"description": "The default redirect URI"
+		  },
+		  "addIns": {
+			"type": "array",
+			"items": {
+			  "type": "object",
+			  "properties": {
+				"id": {
+				  "type": "string",
+				  "description": "Add-in ID"
+				},
+				"type": {
+				  "type": "string",
+				  "description": "Add-in type"
+				},
+				"properties": {
+				  "type": "array",
+				  "items": {
+					"type": "object",
+					"properties": {
+					  "key": {
+						"type": "string",
+						"description": "Add-in property key"
+					  },
+					  "value": {
+						"type": "string",
+						"description": "Add-in property value"
+					  }
+					}
+				  },
+				  "description": "Add-in properties"
+				}
+			  }
+			},
+			"description": "Defines custom behavior that a consuming service can use to call an app in specific contexts"
+		  },
+		  "api": {
+			"type": "object",
+			"properties": {
+			  "acceptMappedClaims": {
+				"type": "boolean",

```

Comment:
@paulculmsee The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. can you please confirm that no users will be affected by this change?

Reply:
@paulculmsee 
the type for this property used to be as string, but now it got changed to boolean.
![image](https://user-images.githubusercontent.com/97551789/218852438-59ea2488-22df-479b-a6ea-95a6972ef467.png)

Reply:
Ah yes - I tested the connector on a reasonable sized tenant with some 1700+ registrations and realised that it was a schema error not picked up on v1. I'll run another test to verify for you to confirm.

Reply:
Confirmed, it should have been boolean. 
![image](https://user-images.githubusercontent.com/4979626/219511535-6d9d7507-324e-4e67-9e9a-d0788dd47b67.png)